ReleaseHistory.md

```diff
@@ -1,4 +1,8 @@
 # SARIF Package Release History (SDK, Driver, Converters, and Multitool)
+## **v4.5.1 UNRELEASED
+* BUG: When the path of the scan target contains invalid characters, report `WRN997_InvalidTarget` before analyzing it against any rule, otherwise `ArgumentException: Illegal characters in path` will be thrown.
+* BUG: When the path of the scan target contains URL Percent-encoding characters, it should proceed with analyzing without throwing `System.IO.FileNotFoundException`.
+
```

Comment:
"Percent" should be lowercase: "percent-encoded"

Reply:
updated